src/libmongoc/tests/unified/operation.c

```diff
@@ -689,10 +689,13 @@ operation_create_find_cursor (test_t *test,
    cursor = mongoc_collection_find_with_opts (
       coll, filter, opts, NULL /* read prefs */);
 
+   ret = true;
+
+   mongoc_cursor_error_document (cursor, &op_error, &op_reply);
+   result_from_val_and_reply (result, NULL, (bson_t *) op_reply, &op_error);
```

Comment:
Is it significant that you swapped the order of `ret = true;` here vs. the original code?

Reply:
No particular reason - want me to move it for consistency?

Reply:
If it was done inadvertently, I think we should leave it in the original form. That would avoid a potential question down the line if someone comes across the diff again.

Reply:
Updated in the last commit.